src/app/rosetta/network.ml

```diff
@@ -29,36 +30,77 @@ module Get_version = [%graphql {|
   }
 |}]
 
-(* TODO: Is there a better way to get the genesis block info *)
+module Get_network =
+[%graphql
+{|
+  query {
+    daemonStatus {
+      peers
+    }
+    initialPeers
+  }
+|}]
+
 let genesis_block_query =
   Caqti_request.find Caqti_type.unit Caqti_type.string
-    "SELECT state_hash FROM blocks WHERE height = 0"
+    "SELECT state_hash FROM blocks WHERE height = 1 LIMIT 1"
```

Comment:
Is the state_hash on block 1 equal to state_hash of block 0? I don't understand why these queries are semantically equivalent

Reply:
height 1 is the genesis block -- the height=0 query that was here before was untested (and broken).

Reply:
ugh I guess it has to be 0 to be consistent with other blockchains, I'll open a bug to fix this